examples/all-clusters-app/all-clusters-common/gen/IMClusterCommandHandler.cpp

```diff
@@ -0,0 +1,2278 @@
+    {
+    case ZCL_BARRIER_CONTROL_CLUSTER_ID:
+        clusters::BarrierControl::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_BASIC_CLUSTER_ID:
+        clusters::Basic::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_BINDING_CLUSTER_ID:
+        clusters::Binding::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_COLOR_CONTROL_CLUSTER_ID:
+        clusters::ColorControl::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_CONTENT_LAUNCH_CLUSTER_ID:
+        clusters::ContentLaunch::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_DOOR_LOCK_CLUSTER_ID:
+        clusters::DoorLock::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_GROUPS_CLUSTER_ID:
+        clusters::Groups::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_IAS_ZONE_CLUSTER_ID:
+        clusters::IasZone::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_IDENTIFY_CLUSTER_ID:
+        clusters::Identify::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_LEVEL_CONTROL_CLUSTER_ID:
+        clusters::LevelControl::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_MEDIA_PLAYBACK_CLUSTER_ID:
+        clusters::MediaPlayback::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_ON_OFF_CLUSTER_ID:
+        clusters::OnOff::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    case ZCL_SCENES_CLUSTER_ID:
+        clusters::Scenes::DispatchServerCommand(apCommandObj, aCommandId, aEndPointId, aReader);
+        return;
+    default:
+        // Unrecognized cluster ID, error status will apply.
+        // TODO: Encode response for Cluster not found
+        break;
+    }
+}
+
+// Cluster specific command parsing
+
+namespace clusters {
+
+namespace BarrierControl {
+
+void DispatchServerCommand(app::Command * command, CommandId commandId, EndpointId endpointId, TLV::TLVReader & dataTlv)
+{
+    {
+        switch (commandId)
+        {
+        case ZCL_BARRIER_CONTROL_GO_TO_PERCENT_COMMAND_ID: {
+            CHIP_ERROR TLVError = CHIP_NO_ERROR;
+            uint8_t percentOpen;
+
+            while ((TLVError = dataTlv.Next()) == CHIP_NO_ERROR)
+            {
+                switch (TLV::TagNumFromTag(dataTlv.GetTag()))
+                {
+                case 0:
+                    TLVError = dataTlv.Get(percentOpen);
+                    break;
+                default:
+                    // Unsupported tag, ignore it.
+                    // TODO: Can be a log for INFO level
```

Comment:
All errors should be logged - I think we can do this instead of just TODO - the complexity does not seem high enough too warrant TODOs